vehicles/src/modules/loa/dto/request/create-loa.dto.ts

```diff
@@ -0,0 +1,109 @@
+import { AutoMap } from '@automapper/classes';
+import { ApiProperty } from '@nestjs/swagger';
+import { Transform } from 'class-transformer';
+import {
```

Comment:
The dto needs to be restructured to something like file: string, body: object.
Bind a custom interceptor on the endpoint that can parse the body using JSON.parse()

Reply:
@praju-aot The request body needs to be multipart/form-data rather than json, and the file could be up to 10MB so I don't think string is a good idea

Reply:
The request body type will remain multipart/form-data, its just the values in the formdata will be restructured. We can discuss.